src/components/ebay-notice/index.js

```diff
@@ -0,0 +1,50 @@
+const processHtmlAttributes = require('../../common/html-attributes');
+const template = require('./template.marko');
+
+
+const constants = {
+    'inline': {
+        headingTag: 'span',
+        contentTag: 'span',
+        mainTag: 'div'
+    },
+    'page': {
+        headingTag: 'h2',
+        contentTag: 'div',
+        mainTag: 'section'
+    }
+};
+
+const defaults = {
+    level: 'page',
+    type: 'confirmation'
+};
+function getInitialState(input) {
+    return input;
+}
+function getTemplateData(state) {
+    const model = {};
```

Comment:
@sharma46bhawana You could also do 
```js
const model = Object.assign({}, defaults, state);
```
and avoid 
```js
model.level = state.level || defaults.level;
model.type = state.type || defaults.type;
```
BTW, the above two lines are repeated twice. Maybe a typo.

Reply:
nice!

Reply:
I will change it

Reply:
Which 2 lines?

Reply:
I think I don't need model anymore

Reply:
Right, you don't need `model` any longer.